### PR TITLE
feat: quit more information page

### PR DESCRIPTION
**Pull request by Nassim-89**
This should be enough for this issue.
As written in the first commit body, we may consider using the previous route instead of linking back to the project - image route if the component ends up being used somewhere else.

NB: my bad for going over the limit of the header size in the first commit, I'll pay more attention to it next time)

### CODE REVIEW

**Review by waliens — CHANGES_REQUESTED** [review]
TL;DR
- changing the cross button by a " < Go back " button
- changing the implementation to use the "go back" approach with vue router

To clarify what is expected exactly, I have just discussed with Gregoire. 

He stated "going back to the viewer" because apparently (though I cannot confirm for sure), this page can only be reached from the viewer currently. In the future, it is not excluded that it could be reached from another page. I suggest that we save ourselves a future "headache" by implementing this fix with a router "go back" rather than a direct link. 

Moreover, in terms of UX, the semantic conveyed by the cross button is a bit odd in this context. When used with a modal, you know you close to go back to what is present behind the modal. In this case, there is no information about what will be revealed when you "close" this page. I suggest to change this by a button " < Go back " where "<" is a left bracket icon. If necessary, the title can be left aligned to fit the button (in case the filename is long).

**Review by waliens — APPROVED** [review]
Awesome, and validated by Grégoire. Let's merge.
